frontend/build.rs

```diff
@@ -48,11 +48,11 @@ pub fn build_types() {
 pub fn build_frontend() {
```

Comment:
Can you make this build script crossplatform? So that it can work on Windows as well